CONTRIBUTING.md

```diff
@@ -1,39 +1,164 @@
+  Some bugs are expressly not good for new contributors, so don't expect 100% overlap between the two.
+  - [Changelog
+      Missing](https://github.com/rails-api/active_model_serializers/issues?q=label%3A%22Changelog+Missing%22+is%3Aclosed) is
+    an easy way to help out.
+
+- If you want to work on new feature development, look for the label [**Feature**](https://github.com/rails-api/active_model_serializers/labels/Feature).
+
+- We are also encouraging comments to substantial changes (larger than bugfixes and simple features) under an "RFC" (Request for Comments) process before we start active development. Look for the [**RFC**](https://github.com/rails-api/active_model_serializers/labels/RFC) label.
 
 ## Issue Labeling
 
 AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIssueLabels) for Github Issues. You can [see our labels here](https://github.com/rails-api/active_model_serializers/labels).
 
-## Contributing
-
-1. Fork it ( https://github.com/rails-api/active_model_serializers/fork )
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Write tests for your feature, or regression tests highlighting a bug
-4. Write the feature itself, or fix your bug
-5. Commit your changes (`git commit -am 'Add some feature'`)
-6. Push to the branch (`git push origin my-new-feature`)
-7. Create a new Pull Request
-8. Update [CHANGELOG.md](https://github.com/rails-api/active_model_serializers/blob/master/CHANGELOG.md)
-with a brief description of any breaking changes, fixes, features, or
-miscellaneous changes under the proper version section.
-9. Iterate on feedback given by the community (fix syntax, modify bits of code, add
+## Submitting a pull request (PR)
+
+1. The vast majority of development is happening under the `master` branch, currently slated for release as `0.10.x`.
+  This is where we would suggest you start.
+1. Fixing bugs is extraordinarily helpful and requires the least familiarity with AMS.
+  Look for issues labeled [**Needs Bug Verification**](https://github.com/rails-api/active_model_serializers/labels/Needs%20Bug%20Verification) and [**Bug**](https://github.com/rails-api/active_model_serializers/labels/bug).
+1. Adding or fixing documenation is also fantastic!
+
+To fetch & test the library for development, do:
+
+1. Fork the repository ( https://github.com/rails-api/active_model_serializers/fork )
+1. `git clone https://github.com/{whoami}/active_model_serializers.git`
+1. `cd active_model_serializers`
+1. `bundle`
+  - To test against a particular rails version, 4.0 is usually the most buggy, set then
+      RAILS_VERSION environment variable as described in the [.travis.yml](.travis.yml).
+      e.g. `export RAILS_VERSION=4.0`.
+1. Create your pr branch (`git checkout -b my-helpful-pr`)
+1. Write tests for your feature, or regression tests highlighting a bug.
+  This is important so AcitveModel Serializers doesn't break it in a future version unintentionally.
```

Comment:
AcitveModel Serializers -> ActiveModel::Serializers
doesn't break it in a future version unintentionally. -> doesn't break it unintentionally in a future version